File: MeetingWithJava/Practice/array/RemoveDuplicate.java
package Practice.array;

public class RemoveDuplicate {

    public static void removeEle(int arr[]){
        int arr2[] = new int[arr.length];
        int index = 0;
        for(int i = 0; i < arr.length; i++){
            for(int j = 0; j < arr.length; j++){
                if(arr[i] == arr[j] && i == j){
                    arr2[index++] = arr[i];
                }
            }
        }

        System.out.println("Updated array: ");
        for(int i = 0; i < index; i++){
            System.out.print(arr2[i]);
        }
    }

    public static void main(String[] args) {
        int arr[] = {1,2,3,2,3,2,3,5,6,9};
        removeEle(arr);
    }
}

File: MeetingWithJS/FINAL_JS_REVESION/JS/Array22222/RemoveDuplicate.js
(function (){
    let arr = [1,3,45,6,8, 3,6];
    let size = arr.length;

    for(let i = 0; i < size; i++){
        for(let j = i+1; j < size; j++){
            if(arr[i] === arr[j]){
                for(let k = j; k < size-1; k++){
                    arr[k] = arr[k+1];
                }
                j--;
                size--;
            }
        }
    }

    for(let i = 0; i < size; i++){
        console.log(arr[i]);
    }
})();